charts/pega/README.md

```diff
@@ -107,7 +107,7 @@ To support this option,
 •  Pass secrets directly to your deployment using your organization's recommend practices.
 
 ##### Things to note in case of providing keystore, certificates for Enabling encryption of traffic between Ingress/LoadBalancer and Pod
-1. Have the CA certificate and keystore as a base64 encrypted string inside the secret manager(AWS Secret Manager, Azure Key Vault etc). [Check this](https://github.com/smootherbug/pega-helm-charts/tree/tomcat-encrypt/charts/pega#enabling-encryption-of-traffic-between-ingressloadbalancer-and-pod)
+1. Have the CA certificate and keystore as a base64 encrypted string inside the secret manager(AWS Secret Manager, Azure Key Vault etc). [Check this](#enabling-encryption-of-traffic-between-ingressloadbalancer-and-pod)
```

Comment:
small tweaks>>
Have the CA certificate and keystore as a base64 encrypted string inside the secret manager(AWS Secret Manager, Azure Key Vault etc). [Check this](#enabling-encryption-of-traffic-between-ingressloadbalancer-and-pod)
to 
Configure the CA certificate and keystore as a base64 encrypted string inside your preferred secret manager (AWS Secret Manager, Azure Key Vault etc). For details, see [this section](#enabling-encryption-of-traffic-between-ingressloadbalancer-and-pod).

Reply:
done